services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -118,10 +128,21 @@ const INDEX_VERIFIED_HEIGHT = 'indexVerifiedHeight';
 
 const validateBlock = (block) => !!block && block.height >= 0;
 
+const getTransactionExecutionStatus = (tx, events) => {
+	// TODO: Update implementation with next SDK release and return with constants from SDK
+	// const expectedEventName = `${transaction.module}commandExecutionResult`;
+	const expectedEventName = `${tx.module}:transaction`;
+	const commandExecResultEvents = events.filter(e => `${e.module}:${e.name}` === expectedEventName);
+	const txExecResultEvent = commandExecResultEvents.find(e => e.topics.includes(tx.id));
+	return txExecResultEvent.data === '0801' ? 'success' : 'fail';
```

Comment:
Maybe we can define a constant for the data `0801`.

Reply:
There's a TODO above to address the same. This most likely will be exposed by the SDK as a constant in the next release. I'd leave it as is for now to avoid rework.